docs/reference/drafttopic.utilities.tests.rst

```diff
@@ -0,0 +1,54 @@
+drafttopic.utilities.tests package
```

Comment:
We probably don't want the tests showing up in our docs.

Reply:
Agreed, will remove.